hydroflow/src/util/unsync/mpsc.rs

```diff
@@ -273,13 +273,14 @@ mod test {
     use tokio::task::LocalSet;
 
     use super::*;
+    use crate as hydroflow;
```

Comment:
@MingweiSamuel is there a cleaner way to do this? the #[hydroflow:main] expands out and contains many references to the hydroflow crate by name, which is not valid here because it is itself inside the hydroflow crate?

Without this line, the below code fails to compile.